docs/hotkeys.md

```diff
@@ -24,7 +24,7 @@
 |Scroll up|<kbd>PgUp</kbd> / <kbd>K</kbd>|
 |Scroll down|<kbd>PgDn</kbd> / <kbd>J</kbd>|
 |Go to bottom / Last message|<kbd>End</kbd> / <kbd>G</kbd>|
-|Trigger the selected entry|<kbd>Enter</kbd>|
+|Trigger the selected entry|<kbd>Enter</kbd> / |
```

Comment:
It looks like we need a special case to handle the documented keys here?

Reply:
Oops with the squashing. Guess I didn't check the diff, my apologies.